code/modules/events/spacevine.dm

```diff
@@ -466,9 +466,10 @@
 		return
 	if(parent)
 		SV.mutations |= parent.mutations
-		var/parentcolor = parent.atom_colours[FIXED_COLOUR_PRIORITY]
-		SV.add_atom_colour(parentcolor, FIXED_COLOUR_PRIORITY)
-		if(prob(mutativness))
+		if(parent.atom_colours)
```

Comment:
this doesn't address the WHY of the parent not having atom colours.

Is it expected or normal that a parent might not have atom colours?

otherwise this is just a bandaid fix